README.md

```diff
@@ -390,45 +390,47 @@ Once we have the translated .po files they can be added to the source code and u
+| EQ_REDIS_PORT                             |                              | Port number of Redis instance used for ephemeral storage                                                       |
+| EQ_DYNAMODB_MAX_RETRIES                   | 5                            |                                                                                                                |
+| EQ_DYNAMODB_MAX_POOL_CONNECTIONS          | 30                           |                                                                                                                |
+| EQ_QUESTIONNAIRE_STATE_TABLE_NAME         |                              |                                                                                                                |
+| EQ_SESSION_TABLE_NAME                     |                              |                                                                                                                |
+| EQ_USED_JTI_CLAIM_TABLE_NAME              |                              |                                                                                                                |
+| WEB_SERVER_TYPE                           |                              | Web server type used to run the application. This also determines the worker class which can be async/threaded |
+| WEB_SERVER_WORKERS                        |                              | The number of worker processes                                                                                 |
+| WEB_SERVER_THREADS                        |                              | The number of worker threads per worker                                                                        |
+| WEB_SERVER_UWSGI_ASYNC_CORES              |                              | The number of cores to initialise when using "uwsgi-async" web server worker type                              |
+| DATASTORE_USE_GRPC                        | False                        | Determines whether to use gRPC for Datastore. gRPC is currently only supported for threaded web servers        |
+| ACCOUNT_SERVICE_BASE_URL                  | `https://surveys.ons.gov.uk` | The base URL of the account service used to launch the survey                                                  |
+| ONS_URL                                   | `https://www.ons.gov.uk`     | The URL of the ONS website where static content is sourced, e.g. accessibility info                            |
+| SDS_API_BASE_URL                          | `http://localhost:5003`      | The base URL of the SDS API used for fetching supplementary data                                               |
+| CIR_API_BASE_URL                          | `http://localhost:5004`      | The base URL of the CIR API used for fetching collection instruments                                           |
```

Comment:
Doc defaults are prod first. Since these aren't defaulted, these dev defaults can be removed.